### PR TITLE
test: add integration test for the rollback cmd

### PR DESCRIPTION
**Pull request by yihuang**
## Description

- check the rollback cmd works in integration tests

depends: https://github.com/cosmos/cosmos-sdk/pull/13089

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: tests/integration_tests/configs/broken-ethermintd.nix [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{ pkgs ? import ../../nix { } }:
```
Comment (yihuang) [inline tests/integration_tests/configs/broken-ethermintd.nix:1]:
```suggestion
{ pkgs ? import ../../../nix { } }:
```

File: tests/integration_tests/configs/broken-ethermintd.nix [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+let ethermintd = (pkgs.callPackage ../../. { });
```
Comment (yihuang) [inline tests/integration_tests/configs/broken-ethermintd.nix:2]:
```suggestion
let ethermintd = (pkgs.callPackage ../../../. { });
```

File: tests/integration_tests/configs/rollback-test.jsonnet [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    }],
+    genesis+: {
+      app_state+: {
+        feemarket: {
+          params: {
+            no_base_fee: true,
+          },
+        },
+      },
+    },
```
Comment (yihuang) [inline tests/integration_tests/configs/rollback-test.jsonnet:16]:
```suggestion
```

**Review by facs95 — APPROVED** [review]
LGTM! Awesome test.. can you please add a changelog?

**Comment by yihuang** [comment]
> LGTM! Awesome test.. can you please add a changelog? 

Oh, I forgot to do that, thanks for the help 👍

**Pull request merged**